competencies/roles/DeveloperLevelMapping.md

```diff
@@ -4,8 +4,10 @@ The three categories of competencies can be used to map to typical job titles sp
 
 | Title/Job           | Key Competencies           | Other Competencies           |     Notes  |
 | ------------------- |:-------------------------:| ----------------------------:| ----------:|
-| SW Dev I (junior)   |                           | 90% emerging; 10% proficient |  New Grad, or first one or two years |
-| SW Dev II           | Proficient: "Tech Design, Implementation and Code Ownership"      |  50% emerging, 50% proficient  |   |
-| SW Dev III (senior) | Proficient: "Communication" and "System Design" | 80% proficient; some authority  | May be expert or generalist | 
-| Principle SW Dev    | Proficient: "Technical Change" | 50% Proficient; 50% Authority | Strong self awareness, Strong on general areas such as business acumen | 
-| Distinguished       | Authority: "Technical Change" | | Broad authority or extreme depth in a technical branch | 
+| SW Dev I (junior)   |   |   |  New Grad, or first one or two years |
+| SW Dev II           |   |  75% emerging, 25% proficient  |   |
+| SW Dev III (intermediate) | Proficient: "Tech Design, Implementation and Code Ownership" | 50% emerging, 50% proficient |   |
+| SW Dev IV (senior I) | Proficient: "Communication" or "System Design" | 50% proficient; some authority  |   |
+| SW Dev V (senior II) | Proficient: "Communication" and "System Design" | 80% proficient; some authority  | May be expert or generalist |
```

Comment:
How do we feel about _any_ emerging in the senior levels?

Reply:
Good question.  I can definitely picture a strong senior dev who is not quite proficient yet in business acumen (particularly if they're new to the business or we haven't provided enough learning opportunities in that area).  I have also worked with senior devs who are technically very capable, but haven't checked off all our proficient points in communication or teamwork (although that's typically indicative of something that needs to be worked on).

Reply:
Perhaps "at least proficient in all role specific"?

Reply:
Made "System Design" proficiency a requirement for any senior level, rather than "all role specific".  We have proficiency in Technical Change for the Principal level, so requiring that here with all the other role specific competencies felt like we would then be setting the bar high on technical skill alone rather than providing opportunity for individuals who don't fit that specific mould.